src/app/pages/sharing/iscsi/fibre-channel-ports/fibre-channel-port/fibre-channel-port.component.ts

```diff
@@ -142,7 +143,7 @@ export class FibreChannelPortComponent implements OnInit {
         this.ws.call('fcport.update', [this.config.id, value]).subscribe(
             (res) => {
                 this.parent.loader.close();
-                this.parent.snackBar.open("Fiber Channel Port " + this.config.name + " update successful.", 'close', { duration: 5000 });
+                this.parent.dialogService.Info(T('Updated'), T('Fiber Channel Port ') + this.config.name + ' update successful.');
```

Comment:
```suggestion
                this.parent.dialogService.Info(T('Updated'), T('Fibre Channel Port ') + this.config.name + ' update successful.');
```
I believe this is the industry standard spelling.